build.sbt

```diff
@@ -23,17 +23,24 @@ ThisBuild / pomExtra := (
 
 publish / skip := true
 
+libraryDependencies += "org.scalatest" %% "scalatest" % "3.0.8" % Test
```

Comment:
We don't have to test the root project, maybe move this into `.jvmSettings()` of crossProject?

Reply:
I add this because `test:compile` will have compilation error. If there is no one use `test:compile`, I can remove it. I add this to prevent the surprise for any new comer. :p

Reply:
As far as I remember, there is another way to avoid this. We use `CrossType.Full` and put souce code into `shared` directory. Then it will not have this issue.

Reply:
How about moving `src` into `minitime` directory? I think we can still use CrossType.Pure after the move?

Reply:
If you meant `minitime/src/main/scala/minitime`, I will give it a try to see the result.